src/sentry/runner/commands/run.py

```diff
@@ -312,6 +312,12 @@ def cron(**options):
     type=int,
     help="How many messages to process (may or may not result in an enqueued task) before committing offsets.",
 )
+@click.option(
+    "--commit-batch-timeout-ms",
```

Comment:
Any particular reason we didn't need this before but we do now? (I was a bit surprised to see we didn't have it earlier)

Reply:
The consumer was a streaming synchronous consumer before the change. Hence it didn't need a timeout. In devserver the commit batch size is 1 so it would commit immediately.